### PR TITLE
[IMP] product_variant_cost_price: when template has only one variant,

### PR DESCRIPTION
**Pull request by oihane**
template standard_price is written twice, creating two entries in history

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by agaldona** [comment]
LGTM :+1:

**Comment by oihane** [comment]
Found more issues... creates more history than it is supposed

File: product_variant_cost_price/models/product_product.py [hunk 1]
```diff
@@ -55,6 +55,12 @@ def _set_standard_price(self, product, value):
 
     @api.model
     def create(self, values):
+        if 'product_tmpl_id' in values and 'standard_price' not in values:
+            template = self.env['product.template'].browse(
+                values.get('product_tmpl_id'))
+            values.update({
```
Comment (pedrobaeza) [inline product_variant_cost_price/models/product_product.py:61]: Why do you need this?

Comment (oihane) [inline product_variant_cost_price/models/product_product.py:61]: when you create a "new" variant using, create_variants_ids method from product.template the taken price is always 0.0 not the one in the template, that way it creates 2 history lines.
and it was impacting with the price in the template, changing it to 0.0

Comment (pedrobaeza) [inline product_variant_cost_price/models/product_product.py:61]: OK, I see, so it's not a problem of the history, but the variant cost price. Thanks for clarifying.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by oihane** [comment]
@agaldona can you take another look, please.

**Comment by agaldona** [comment]
Reviewed :+1:

**Pull request merged**